frontend/components/Blocks/SectionBlock/SectionBlock.tsx

```diff
@@ -233,6 +272,7 @@ export default function SectionBlock({ data }: Props) {
 
   return (
     <div className={`${backgroundFullcolor} `}>
+      <ChallengeFeedbackModal feedbackmodals={feedbackmodals} kpis={kpis} content={content} />
```

Comment:
Ik krijg hier een foutmelding wanneer ik een storyline open ipv een challenge omdat deze geen feedbackmodals heeft

```suggestion
      {feedbackmodals && <ChallengeFeedbackModal feedbackmodals={feedbackmodals} kpis={kpis} content={content} />}
```